src/components/CommentCard.jsx

```diff
@@ -0,0 +1,20 @@
+
+
+const CommentCard = ({comment})=>{
+return(
+    <article className="CommentCard">
+        <ul>
+            <li>{comment.body}</li>
+            <li>{comment.votes}</li>
+            <li>{comment.created_at}</li>
```

Comment:
It would be better to have the timestamp formatted in a bit more of a human-readable way, than the default. Have a look into some packages that can do it for you 😄